cumulusci/version.txt

```diff
@@ -1 +1 @@
-3.51.1
+3.51.1.dev1
```

Comment:
This should be 3.52.0.dev0 -- 3.51.1.dev1 comes before 3.51.1 the way that PyPI sorts version numbers.